pkg/adapter/awssqs/adapter_test.go

```diff
@@ -52,7 +51,7 @@ func TestPostMessage_ServeHTTP(t *testing.T) {
 			defer sinkServer.Close()
 
 			a := &Adapter{
-				QueueUrl:             "https://test.sqs.aws/123123",
+				QueueURL:             "https://test.sqs.aws/123123",
```

Comment:
Because we now extract the region from the URL, let's make this more like a real URL, e.g. `https://sqs.us-east-2.amazonaws.com/123456789012/MyQueue`.

Reply:
https://github.com/knative/eventing-sources/pull/124/commits/573ca8b6814d24cd42ef3c01f50a7ec7bd76106b